From 2c8d8d1d93471a7055c7c9f4699bda4c7de80662 Mon Sep 17 00:00:00 2001
From: boombatower <jimmy@boombatower.com>
Date: Thu, 23 Jun 2016 00:32:03 -0500
Subject: [PATCH] Invoke libstdc removal before starting steam.

---
 steam | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/steam b/steam
index a4d519d..ee5d164 100755
--- a/steam
+++ b/steam
@@ -3,6 +3,8 @@
 # This is the Steam script that typically resides in /usr/bin
 # It will create the Steam bootstrap if necessary and then launch steam.
 
+steamruntime-fix
+
 # Get the full name of this script
 export STEAMSCRIPT="$(cd "${0%/*}" && echo "$PWD")/${0##*/}"
 
-- 
2.8.3

