From 8a02a97757127a529178397647cbd71bc769d5bc Mon Sep 17 00:00:00 2001
From: boombatower <jimmy@boombatower.com>
Date: Sat, 8 Oct 2016 23:19:22 -0500
Subject: [PATCH] Invoke steamtricks if available.

---
 steam | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/steam b/steam
index 1b71982..847ee0d 100755
--- a/steam
+++ b/steam
@@ -1,4 +1,13 @@
 #!/usr/bin/env bash
+if ! test "$NO_STEAMTRICKS" == 1 && [ -x "$(which steamtricks)" ] ; then
+  # does not seem to be a better place to daemon-reload since install time runs
+  # as root and does not have user context available
+  systemctl --user daemon-reload
+  systemctl --user import-environment DISPLAY
+  systemctl --user import-environment XDG_CURRENT_DESKTOP
+  systemctl --user start steamtricksd
+fi
+
 #
 # This is the Steam script that typically resides in /usr/bin
 # It will create the Steam bootstrap if necessary and then launch steam.
-- 
2.10.0

